backend/canvas_app_explorer/management/commands/manage_pylti.py

```diff
@@ -0,0 +1,66 @@
+from Crypto.PublicKey import RSA
+from Crypto.PublicKey.RSA import RsaKey
+from django.core.management.base import BaseCommand, CommandParser
+from django.db.utils import IntegrityError
+
+# https://github.com/dmitry-viskov/pylti1.3/blob/master/pylti1p3/contrib/django/lti1p3_tool_config/models.py
+from pylti1p3.contrib.django.lti1p3_tool_config.models import LtiToolKey, LtiTool
+
+
+class Command(BaseCommand):
+    help = "Generate a private/public key pair and add them to the pylti13 database. This can be called with the same issuer + client ID to update a previous key. The Tool Key will be re-used"
+
+    def add_arguments(self, parser: CommandParser):
+        parser.add_argument('--platform', dest='platform', type=str,
+                            help="Canvas Platform for LTI Launch, with or without https://",
+                            default="canvas.instructure.com")
+        parser.add_argument('--client_id', dest='client_id', type=int, required=True, help="Canvas LTI Client ID")
+        parser.add_argument('--title', dest='title', required=True, type=str, help="LTI Title")
+        parser.add_argument('--tool_key', dest='tool_key', required=True, type=str,
+                            help="Name of Tool Key to use, will create if new")
+        parser.add_argument('--deployment_ids', dest='deployment_ids',
+                            nargs='*', type=str, help="List of Deployment ID(s). Can be multiple.", default="")
+
+    def handle(self, *args, **options: dict):
+        key: RsaKey = RSA.generate(4096)
+
+        self.stdout.write('Generating public and private key for LTI Tool Key table')
```

Comment:
Maybe "LTI Tool Key record"?

Reply:
Some other messages about the other steps may be appropriate.

Reply:
I can just drop the table part, will add a few other messages